File: 2026-09/src/day_10/prms_120864/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(string my_string) {
    for (char& c : my_string) {
        if (!isdigit(c)) c = ' ';
    }

    stringstream ss(my_string);
    int sum = 0;
    int x;
    while (ss >> x) sum += x;

    return sum;
}

File: 2026-09/src/day_10/prms_120864/Solution.java
package day_10.prms_120864;

import java.util.*;

class Solution {
    public int solution(String my_string) {
        char[] arr = my_string.toCharArray();
        for (int i = 0; i < arr.length; i++) {
            if (!Character.isDigit(arr[i])) arr[i] = ' ';
        }

        StringTokenizer st = new StringTokenizer(new String(arr));
        int sum = 0;
        while (st.hasMoreTokens()) {
            sum += Integer.parseInt(st.nextToken());
        }

        return sum;
    }
}
